updates links and descriptions to Protect Your Data activities

hannahkane opened the pull request:
- [X] Localization addressed - NA

Fixes #2392

**Changes proposed in this pull request:**
- updates links and index copy for Privacy Basics module
- updates two images

hannahkane commented:
anyone have a moment to review this? @alanmoo? @gideonthomas?

alanmoo commented:
Looking now

alanmoo commented on pages/protect-your-data.jsx:
```diff
@@ -24,73 +23,61 @@ var LearningObjectives = (
-    title: "Participating on the Web",
-    activities: [
+        title: "Create Secure Passwords",
+        image1x: "/img/pages/protect-your-data/pass-phrase.jpg",
```
I don't know where this image came from, but 
![](https://media.giphy.com/media/dOJt6XZlQw8qQ/giphy.gif)

alanmoo commented on pages/protect-your-data.jsx:
```diff
@@ -24,73 +23,61 @@ var LearningObjectives = (
-        description: "Learners become privacy mentors to their peers by sharing their expertise through discussion or creating their own &ldquo;recipe&rdquo; documentation on privacy.",
-        link: "http://mozilla.github.io/webmaker-curriculum/ProtectingYourData/session03-privacycoach.html"
+        subtitle: "Open Practice, Protect",
+        description: "rYou will make a teaching and learning resource you can use to mentor your peers about online privacy and protecting their data while learning web literacy skills like <strong>open practice<strong> and <strong>protect</strong>.",
```
Extra r

alanmoo commented on pages/protect-your-data.jsx:
```diff
@@ -6,14 +6,13 @@ var ActivitySection = require('../components/activity-section.jsx');
-      src2x="/img/pages/protect-your-data/photo-protect-your-data@2x.png"
-      alt="">
-        <h2>Engage learners around the topic of privacy in this six-part module, developed in conjunction with Hive Toronto and the Mozilla Privacy Team.</h2>
+      src1x="/img/pages/protect-your-data/ip-tracer-badge.jpg"
```
This image gets squished

hannahkane commented:
Thanks, @alanmoo. I fixed the typo and added the style. I think it looks okay. Most obvious repercussions are on /activities, but I think it's...fine. What do you think?

alanmoo commented:
Good point. I think removing [this line](https://github.com/mozilla/learning.mozilla.org/blob/master/less/components/illustration.less#L55) should reduce that weirdness, though I didn't check everywhere the `<Illustration>` component might be used.

alanmoo commented:
Ok, if you merge in https://github.com/hannahkane/learning.mozilla.org/pull/1, this PR should then be ready to go.

hannahkane commented:
Thanks @alanmoo!
